ros2bag/ros2bag/verb/info.py

```diff
@@ -47,14 +47,14 @@ def _is_service_event_topic(self, topic_name, topic_type) -> bool:
         return True
 
     def main(self, *, args):  # noqa: D102
-        m = Info().read_metadata(args.bag_path, args.storage)
-        if args.topic_name:
-            for topic_info in m.topics_with_message_count:
-                if not self._is_service_event_topic(topic_info.topic_metadata.name,
-                                                    topic_info.topic_metadata.type):
-                    print(topic_info.topic_metadata.name)
+        if args.verbose:
```

Comment:
How about to output a warning while users set '-t' and '-v' at the same time ?
 
 ```python
 if args.topic_name and args.verbose :
     print("Warning！You have set both the '-t' and '-v' parameters. The '-t' parameter will be ignored.")
```

Reply:
@Barry-Xu-2018 Sorry for my late response. Yeah, with warning will be even better. :+1:

Reply:
Okay. I added a new commit https://github.com/ros2/rosbag2/pull/1480/commits/ab62249b08e2ba497b1928eaacad8876437c8610.